README.rst

```diff
@@ -64,7 +64,6 @@ which can seen by running ``sphinx-autobuild --help``:
      --ignore IGNORE       glob expression for files to ignore, when watching for changes
      --no-initial          skip the initial build
      --open-browser        open the browser after building documentation
-     --delay DELAY         how long to wait before opening the browser
```

Comment:
Could maybe leave the option in, but just ignore it (or print a note that it can be removed from the call site).